ipatests/test_integration/test_external_ca.py

```diff
@@ -279,3 +279,77 @@ def test_install_external_ca(self):
         # Install new cert
         self.master.run_command([paths.IPA_CACERT_MANAGE, 'install',
                                  root_ca_fname])
+
+
+class TestMultipleExternalCA(IntegrationTest):
+    """setup ext-ca1 install ipa-server with ca1
+     setup ext-ca2 renew ipa-server with ext-ca2
```

Comment:
Sounds a little confusing. Could you re-phrase pls?

Reply:
In commit it is changed to "Setup externally signed ca1
     install ipa-server with with externally signed ca1
     Setup externally signed ca2 and renew ipa-server with
     externally signed ca2 and check the difference in certificate"